acceso/urls.py

```diff
@@ -28,4 +28,5 @@
         name='db_list_json',
     ),
     path('documentos/', views.documentos, name='documentos'),
+    path('index2/', views.index2, name='index2'),
```

Comment:
index2 doesn't describe the function of this page.  It should be something like "filtrar_imagenes"